setup.cfg

```diff
@@ -145,10 +145,13 @@ ignore_errors=True
 [mypy-pandas.tests.extension.json.test_json]
 ignore_errors=True
 
+[mypy-pandas.tests.indexes.datetimes.test_tools]
```

Comment:
xref #28932, add `#type: ignore` as an alternative to ignoring all errors in module

Reply:
a 'cleaner' fix in https://github.com/pandas-dev/pandas/pull/28339/commits/98491632b70ae319afbb14c04e5740f06bedd89e, that could be simply reverted when `--warn-unused-ignores` reports issue fixed.